dgea/dgea_scvi.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+import scanpy as sc
+import random
+import anndata as ad
+import os
+
+import torch
+from scvi.model import SCANVI, SCVI
+
+def scanvi_dgea(adata:ad.AnnData, groupby:str, reference:str, alternative:str):
+     
+    directory_model = "data/"
+    model_path = os.path.join(directory_model, "model.pt")
+    print(reference, alternative)
+    
+    SCANVI.prepare_query_anndata(adata = adata, reference_model=directory_model)
```

Comment:
We do not necessarily receive an SCANVI model, in some cases it might also be an SCVI model (if no cell type annotations where available in the SIMBA input)